homeassistant/components/huawei_lte/__init__.py

```diff
@@ -40,6 +59,34 @@
 
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
 
+DEVICE_TRACKER_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=DEVICE_TRACKER_KEYS): vol.All(
```

Comment:
We don't allow monitored conditions in config anymore.
https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

We can implement support for disabling entities instead.

https://developers.home-assistant.io/docs/en/entity_registry_disabled_by.html

Reply:
ADR-0003 does not apply, it explicitly says "if that data is fetched in a single API request". The whole point here is that every category requires a separate API call/HTTP request.

Reply:
Ok. Can we solve that problem by using the entity disable options instead?

Reply:
I don't know. And wrt disabled by default, see my comment below.